### DISCUSSION TITLE
NetBox activities at FOSDEM 2024

### DISCUSSION THREAD

**Discussion by jeffgdotorg** [netbox-community/netbox#14701] [Announcements]
[Unfortunately this event has been canceled due to a family emergency. If you registered for the meetup, you should hear from the NetBox Labs marketing / events team very soon with follow-up.]

It's the time of year again when people in the know get excited to come to [FOSDEM](https://fosdem.org/2024/), the annual community-organized meetup for FLOSS hackers and users across Europe. Attending FOSDEM is a special treat for me, and I'm pleased to announce that this year I'll be giving a [talk](https://fosdem.org/2024/schedule/event/fosdem-2024-3159-embiggen-the-box-extending-netbox/) about NetBox extensibility in the [Network devroom](https://fosdem.org/2024/schedule/track/network/).

NetBox Labs will also sponsor [a meetup for NetBox users](https://netboxlabs.com/events/fosdem-24/), and anybody who's NetBox-curious, on Saturday evening. Please register your interest at the link above to receive details on the time and place as soon as it's confirmed.

I hope to see you there!

**Comment by peteeckel** [comment]
> one evening after the conference

Does that mean on Sunday evening? That would be quite unfortunate, in my experience most FOSDEM participants leave on Sunday after the last talks, and there's the issue of Monday being a regular working day.

**Comment by jeffgdotorg** [comment]
I’m aiming for Saturday evening. I agree with your points about Sunday.

**Comment by jeffgdotorg** [comment]
The meetup is now penciled in for Saturday evening in Ixelles, pending final confirmation from the venue manager. Please [register](https://netboxlabs.com/events/fosdem-24/) to help us estimate the number of attendees!

**Comment by yaiqsa** [comment]
Unfortunately I won't be able to attend this year(I would have joined both sessions otherwise!). A friend of mine will go though, I'll send him to the meeting.

However, a more important matter for an event like FOSDEM: Will there be NetBox stickers?

**Reply by jeffgdotorg** [reply]
Of course there will be NetBox stickers!

**Comment by jeffgdotorg** [comment]
I've had to cancel my trip due to a family medical emergency, so I'm sad to say that this meetup will not be happening after all. If you registered to attend, you can expect an update soon from our marketing and events team. I hope to see you all at another event in the near future!
